barretenberg/cpp/pil/vm2/context.pil

```diff
@@ -1,17 +1,79 @@
 // This is a virtual gadget, which is part of the execution trace.
 namespace execution;
 
+    // Useful to define some opcodes within this pil file
+    // Constrained to be boolean by execution instruction spec table
+    pol commit call_sel;
```

Comment:
Won't you also have this in the execution trace eventually?

Reply:
this is actually the one from the execution trace..but you are right that perhaps i should move this to the execution.pil rather than this virtual trace.

I had it in here because my intuition is that most of the relations involving `call_sel` will be in this file.